lib/Comment/styles.scss

```diff
@@ -0,0 +1,56 @@
+.conversation__text__cutoff {
+  &:after {
+    @apply absolute h-8 w-full;
+    content: ' ';
+    left: 40px;
+    top: 96px;
+    background: linear-gradient(
+      to bottom,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(253, 253, 253, 0.5) 50%,
+      #fff 100%
+    );
+  }
+}
+
+.conversation__inactive {
+  &:hover {
+    .conversation__text__cutoff {
+      &:after {
+        background: linear-gradient(
+          to bottom,
+          rgba(255, 255, 255, 0) 0%,
+          rgba(253, 253, 253, 0.5) 50%,
+          #f0f2f5 100%
+        );
+      }
+    }
+  }
+}
+
+.new-comment__input {
+  @apply border-0 p-2;
+}
+
+.edit-comment__input {
+  @apply border-0 pr-2 py-0 pl-0;
+}
+
+.conversation-comment__line {
+  &:before {
+    @apply absolute h-full border-l-2 border-solid border-neutral-90 border-r-0 border-b-0;
+    content: ' ';
+    top: 38px;
+    left: 22px;
+  }
+}
```

Comment:
This is the line that joins up the avatars. Tailwind doesn't like pseudo elements + `top left bottom right` values, hence it's here.

Reply:
Are we getting rid of these anyway in the new designs?

Reply:
https://www.npmjs.com/package/tailwindcss-pseudo-elements there's a plug in that will apparently give better support for pseudo elements or this is probably config we can add in if we think its worth it (same with top/left/bottom/right values)

Reply:
> Are we getting rid of these anyway in the new designs?

The lines still appear on collapsed conversations